src/main/features/response/routes/reject-all-of-claim.ts

```diff
@@ -67,7 +68,11 @@ export default express.Router()
             res.redirect(Paths.sendYourResponseByEmailPage.evaluateUri({ externalId: externalId }))
             break
           case RejectAllOfClaimOption.ALREADY_PAID:
-            res.redirect(Paths.defendantHowMuchPaidClaimant.evaluateUri({ externalId: externalId }))
+            if (FeatureToggles.isEnabled('admissions')) {
```

Comment:
That only checks config, please use one that checks opted in features on claim. See `ClaimFeatureToggles.areAdmissionsEnabled`